src/components/DropZone/DropZone.scss

```diff
@@ -186,6 +184,10 @@ $dropzone-stacking-order: (
     }
   }
 
+  .Container {
+    @include focus-ring($border-width: border-width(thick));
+  }
+
   &.Overlay {
     // Same calculation used for focus-ring
     border-radius: calc(var(--p-border-radius-base) + 3px - 1px + #{rem(1px)});
```

Comment:
I don't think we need the `- 1px + #{rem(1px)}` since this will be fixed/consistent?

Reply:
Agreed, this math is pretty weird. Between the mixed `rem/px` units, and subtracting and adding `1`, this could be cleaned up a lot

Reply:
Good catch. I'll track that issue here among other focus ring related issues: https://github.com/Shopify/polaris-ux/issues/396